BUFR_TableD_en_40.csv

```diff
@@ -1003,3 +1003,36 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 40,Additional satellite report sequences,340026,(Lidar observations from satellite),,015073,Attenuated backscatter,,,,Operational
 40,Additional satellite report sequences,340026,(Lidar observations from satellite),,015064,Uncertainty in attenuated backscatter,,,,Operational
 40,Additional satellite report sequences,340026,(Lidar observations from satellite),,201000,Change data width,Cancel,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,008070,Vertical sounding product qualifier,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,001033,Identification of originating/generating centre,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,001034,Identification of originating/generating sub-centre,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,001007,Satellite identifier,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,002019,Satellite instrument,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,005040,Orbit number,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,025075,Satellite antenna corrections version number,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,201133,Change data width,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,005041,Scan line number,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,201000,Change data width,Cancel,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,005043,Field of view number,,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,301011,"Year, month, day",,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,301013,"Hour, minute, second",,,,Operational
+40,Additional satellite report sequences,340028,(Satellite sounding data),,201133,Change data width,,,,Operational
```

Comment:
Is 005001 and 006001 (latitude and longitude) missing or is issue #156 out of date?

Reply:
@amilan17 Many thanks for spotting this, I have updated the branch with 005001 and 006001 (latitude and longitude) as in issue https://github.com/wmo-im/BUFR4/issues/156